lms/templates/problem.html

```diff
@@ -30,12 +30,12 @@ <h3 class="hd hd-2 problem-header" id="${ short_id }-problem-title" aria-describ
         <div class="problem-action-buttons-wrapper">
         % if demand_hint_possible:
         <span class="problem-action-button-wrapper">
-            <button class="hint-button problem-action-btn btn-default btn-small" data-value="${_('Hint')}" ${'' if should_enable_next_hint else 'disabled'}><span class="icon fa fa-question" aria-hidden="true"></span>${_('Hint')}</button>
+            <button type="button" class="hint-button problem-action-btn btn-default btn-small" data-value="${_('Hint')}" ${'' if should_enable_next_hint else 'disabled'}><span class="icon fa fa-question" aria-hidden="true"></span>${_('Hint')}</button>
```

Comment:
This change was necessary because of #13422. FYI @muzaffaryousaf @muhammad-ammar and @cptvitamin